src/plugins/plot/MctTicks.vue

```diff
@@ -210,8 +215,8 @@ export default {
             if (this.shouldRegenerateTicks(range, forceRegeneration)) {
                 let newTicks = this.getTicks();
                 this.tickRange = {
-                    min: Math.min.apply(Math, newTicks),
-                    max: Math.max.apply(Math, newTicks),
```

Comment:
Our coverage tool has recently been updated, we can freely use modern syntax now.